go.mod

```diff
@@ -5,6 +5,7 @@ require (
 	github.com/afex/hystrix-go v0.0.0-20180502004556-fa1af6a1f4f5
 	github.com/aliyun/alibaba-cloud-sdk-go v0.0.0-20190802083043-4cd0c391755e // indirect
 	github.com/apache/dubbo-go-hessian2 v1.2.5-0.20191029001541-894e45c9aaaa
+	github.com/bmizerany/assert v0.0.0-20160611221934-b7ed37b82869
```

Comment:
I do not think this so old package is qualified which is not been maintained. maybe u should use https://github.com/stretchr/testify/assert.

Reply:
it have been deleted